Nodejs/Product/TestAdapter/TestFrameworks/Jest/jest.js

```diff
@@ -116,5 +123,36 @@ function logError() {
     console.error.apply(console, errorArgs);
 }
 
+function readConfigs(projectFolder, context)
+{
+    // First look for Jest.config.js, otherwise look at the package.json under "jest" tag
+    var userConfig;
+    const jestConfigPath = projectFolder + "\\jest.config.js";
+    const packageJsonPath = projectFolder + "\\package.json";
+
+    if(fs.existsSync(jestConfigPath))
+    {
+        userConfig = require(jestConfigPath);
+        mergeConfigs();
+        return userConfig;
+    }
+
+    if(fs.existsSync(packageJsonPath))
+    {
+        userConfig = require(packageJsonPath).jest;
+        mergeConfigs();
+        return userConfig;
+    }
+
+    function mergeConfigs()
+    {
+        // If no config was found OR the user doesn't have these tags set up, add it.
+        if(!userConfig) userConfig = {};
```

Comment:
You could also use the logical assignment `||=`. It's supported since node 15.14.0, but I don't remember how far back do we support as well.